docs: add docs on bg sync (#11)
jina-ai/executor-hnsw-postgres: README.md

)
```

### Automatic background syncing

**⚠ WARNING: Experimental feature**

Optionally, you can enable the option for automatic background syncing of the data into HNSW.
This creates a thread in the background of the main operations, that will regularly perform the synchronization.
This can be done with the `sync_interval` constructor argument, like so:

```python
Flow().add(
    uses='jinahub://HNSWPostgresIndexer',
    uses_with={'sync_interval': 5}
)
```

> This is an integer that represents the amount of seconds to wait between synchronization attempts.

Reviewer: I think it would be good if you can mention what `this` is? Even though it is the continuation of the previous statement yet the above code snippet breaks the reading flow for me. But it is just my perspective, you can ignore it if it doesn't make sense.
```suggestion
`sync_interval` argument accepts an integer that represents the amount of seconds to wait between synchronization attempts.
```

Author: makes sense. applied
